src/olympia/users/tests/test_user_utils.py

```diff
@@ -5,25 +5,11 @@
 
 from olympia.amo.tests import TestCase
 from olympia.users.models import BlacklistedName, UserProfile
-from olympia.users.utils import (
-    autocreate_username, EmailResetCode, UnsubscribeCode)
+from olympia.users.utils import autocreate_username, UnsubscribeCode
 
 
 class TestEmailResetAndUnsubscribeCode(TestCase):
```

Comment:
In fact you can probably make the only method left a function and get rid of the class.